lib/json/pure/generator.rb

```diff
@@ -330,10 +331,13 @@ def json_transform(state)
                 result << %{"#{String(value)}"}
               end
               first = false
+              count += 1
             }
             depth = state.depth -= 1
-            result << state.object_nl
-            result << state.indent * depth if indent
+            unless count.zero?
```

Comment:
Could we use `unless first` to handle this instead of adding `count`?

Reply:
I wasn't quite sure about what the 'object' could be and wether it was possibly an enumerator that can only run once?

Reply:
`first == true` is always equivalent to `count == 0`, isn't it? (they're modified together in the current diff)

Reply:
Ah I see what you mean! Yes I'll do that.

Reply:
Done - I thought you were suggesting I called `.first` on the contents.